ec/Cargo.toml

```diff
@@ -14,25 +14,25 @@ edition = "2021"
 rust-version = "1.57"
```

Comment:
@Pratyush just a quick check, this means that our minimal support version is 1.64 right?

Reply:
Yup

Reply:
```suggestion
rust-version = "1.64"
```